### ISSUE TITLE
Does not work in 1.0.7

### ISSUE THREAD

**Issue by mikedoth** [lwblackledge/file-watcher#3]

**Comment by lwblackledge** [comment]
Hi @mikedoth, can you be a bit more specific please? I tested in the latest version I have (1.0.10) and it was working as expected. Can you check for any error messages in the dev console?

**Comment by mikedoth** [comment]
Atom claims 1.0.7 is the latest version and will not update beyond that. I
opened a test document in dev mode, then the same document in notepad. I
edited the notepad file and saved it and atom just updates without throwing
any console errors  or notifications. Also I have the box checked in the
extension.

**Comment by dpanter** [comment]
Using version 1.0.11 on Linux Mint Debian Edition with KDE 4, this bug happens here as well.
Opened file in Atom, open same file in another editor, add some text and save file.
Atom instantly updates the file, new text appears and no message what so ever appears in Atom, nothing appears in the dev console either.

**Comment by lwblackledge** [comment]
This is a duplicate of issue #2. This package just hooks into the conflict event raised by `atom\text-buffer`, which does not fire an event if the file changes and there are _no_ changes pending within the atom editor.

I am looking into adding this as an option but so far I have been unsuccessful.

**Comment by mikedoth** [comment]
My coworker helped me find the proper error, and fyi I am on windows.
'diff' is not recognized as an internal or external command, ooperable
program or batch file.

**Comment by lwblackledge** [comment]
Interesting, you might have a deeper problem with your Atom install.
I do not use `diff` anywhere in `file-watcher` but it is used in [text-buffer](https://github.com/atom/text-buffer/blob/master/src/text-buffer.coffee) for a couple of operations.
Can you try a clean install or Atom? You might need to raise an issue for atom or text-buffer directly.

However, the scenario you described in [this comment](https://github.com/lwblackledge/file-watcher/issues/3#issuecomment-137498511) is the same as issue #2, and not something that I can fix right now since it involves intercepting or overriding the [text-buffer](https://github.com/atom/text-buffer/blob/master/src/text-buffer.coffee) code.

**Comment by mikedoth** [comment]
Disregard my last comment.. I confused this problem with another. I was trying to get atom-cli-diff to work property. As for this package, this morning I did a reinstall and have yet to see any new errors that would indicate it's broken, but it still does not work.

**Comment by lwblackledge** [comment]
Hi @mikedoth,
The scenario you described [here](https://github.com/lwblackledge/file-watcher/issues/3#issuecomment-137498511) is actually not what this package was created to do:

Atom by default will reload a file if you have **not** made any changes. This is controlled by [text-buffer](https://github.com/atom/text-buffer/blob/master/src/text-buffer.coffee) and something I am working on as part of issue #2.

I made this originally to show a prompt if you **have** made changes and something changes the file on disk. By default Atom will just ignore disk changes and not show a prompt.

Consider this scenario:
1.  Open a document in Atom
2.  Add text (unsaved changes)
3.  Open the same document in notepad
4.  Add text in notepad and save
5.  Check Atom and see that the text from step 2 is still there 

I created this package to add a prompt in atom between step 4 and 5 because I want to know if something changes the file I'm working on.

If you **do not** add text at step 2, at step 5 you will see that Atom reloaded the file because there were _no_ unsaved changes. 

I hope that makes this clearer.
